Source/CarthageKit/Git.swift

```diff
@@ -394,6 +394,10 @@ public func resolveReferenceInRepository(repositoryFileURL: NSURL, reference: St
 /// Attempts to determine whether the given directory represents a Git
 /// repository.
 public func isGitRepository(directoryURL: NSURL) -> SignalProducer<Bool, NoError> {
+	if !NSFileManager.defaultManager().fileExistsAtPath(directoryURL.path!) {
+		return SignalProducer(value: false)
+	}
+
```

Comment:
I actually like it here. I'd rather make callers check that it doesn't exist if they care.